### PR TITLE
cleanup CRA code we don't need

### PR DESCRIPTION
**Pull request by Reinoptland**
# What this pull request does

- Removed some initial code we will not use
- Remove CSS files, we style the app using SASS instead

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,5 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return <div></div>;
```
Comment (sjaak1992) [inline src/App.js:2]: What about "hello world?"

**Review by sjaak1992 — COMMENTED** [review]
Looks very good Rein!

**Review by ShaRosaW — APPROVED** [review]
Approve!

**Comment by Reinoptland** [comment]
It's not much but it is honest work

**Pull request merged**